addons/website_discuss_room/models/chat_room.py

```diff
@@ -53,9 +50,21 @@ def _compute_is_full(self):
             else:
                 room.is_full = room.participant_count >= int(room.max_capacity)
 
-    def _compute_jitsi_server_domain(self):
-        jitsi_server_domain = self.env['ir.config_parameter'].sudo().get_param(
-            'website_jitsi.jitsi_server_domain', 'meet.jit.si')
-
+    @api.depends("discuss_channel_id.rtc_session_ids")
+    def _compute_participant_count(self):
         for room in self:
-            room.jitsi_server_domain = jitsi_server_domain
+            room.participant_count = len(room.discuss_channel_id.rtc_session_ids)
+
+    def get_room_url(self):
+        self.ensure_one()
+        if not self.discuss_channel_id:
+            self.create_new_discuss_channel()
+        return self.discuss_channel_id.invitation_url
+
+    def create_new_discuss_channel(self):
+        self.ensure_one()
```

Comment:
Was thinking about that one if called on a room that already have a discuss channel.
So instead I think it would be a good idea to have a method that get or create the channel (will also make it private).
Something like this:
```suggestion
        self.ensure_one()
        channel = self._get_or_create_discuss_channel()
        return channel.invitation_url

    def _get_or_create_discuss_channel(self):
        self.ensure_one()
        if self.discuss_channel_id:
            return self.discuss_channel_id
        ...
```